src/tests/missions.test.js

```diff
@@ -0,0 +1,34 @@
+import {
+  waitFor, screen, render, fireEvent,
+} from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../redux/configureStore';
+import Missions from '../components/Missions';
+
+describe('Testing the Missions component', () => {
+  test('When data fetched from API all 10 missions are rendering succesfully', async () => {
+    render(<Provider store={store}><Missions /></Provider>);
+    await waitFor(() => {
+      expect(screen.getAllByText('Join mission').length).toBeGreaterThanOrEqual(10);
+    });
+  });
```

Comment:
- [ ] Some tests are failing. Kindly fix this issue.
![Screenshot from 2022-06-24 03-05-49](https://user-images.githubusercontent.com/65860678/175439807-81bab574-db0c-450d-891c-ca64fd39949c.png)

Reply:
This is not yet implemented as requested, please fix the error.

![image](https://user-images.githubusercontent.com/55635977/175571987-553e8d29-3e11-44f9-8664-9d5672ba79a1.png)